js/templates/modals/wallet/transaction.html

```diff
@@ -65,8 +65,11 @@
       <div class="" style="flex-shrink: 0;max-width: 80px">
         <div class="noOverflow"><a class="clrT2 js-txidLink"><%= ob.txid %></a></div>
       </div>
-      <div>
-        <div class="noOverflow"><%= ob.translatedMemo %></div>
+      <%
+        const memo = ob.translatedMemo || ob.memo.slice(0, 300);
```

Comment:
The "untruncated" version is shown in a tooltip. Actually, I do cut it off at 300 so it doesn't overwhelm the layout. I don't think the server limits the size, so I want to prevent some ridiculous size from being shown. I would think 300 is big enough for 99.9% of notes.